Content updates (#92)
alphagov/accessibility-personas: source/ashleigh/index.html.md

* press 'Tab' to go to the next link or form element
* press 'Shift + Tab' to go back to the previous link or form element
* press 'Search + Arrow Right/Left' to read text in between those (or 'Shift + Alt + Arrow Right/Left' on Windows or 'Cmd + Ctrl + Arrow Right/Left' on Mac)

> * learn more [shortcuts for Chromebooks](https://support.google.com/chromebook/answer/7031755) and [shortcuts for the Chrome extension](http://web.archive.org/web/20200121050553/https://www.chromevox.com/keyboard_shortcuts.html)

Reviewer: ```suggestion
* learn more [shortcuts for the ChromeVox screen reader on Chromebooks](https://support.google.com/chromebook/answer/7031755)
```
I feel that if we're talking about a specific screen reader then we should mention it by name, and I don't really see the value in including an archived link to a Chrome extension that's not supported any more.

Also wondering if there's a specific reason this page uses ChromeVox - would it make more sense to use a more widely-available screen reader? That's probably a question for a different pull request though.

Author: There are currently two implementations for Ashleigh:

* the original one uses the native ChromeVox in a Chromebook
* the one we use for remote sessions uses the ChromeVox extension in a Chrome browser

We don't have the option to use a more widely-available screen reader. ChromeVox is the only option on a Chromebook. And the ChromeVox extension is the only option when using browser profiles.

The value of the archived link is that it's the only place where the shortcuts for the ChromeVox extension are listed. As long as the extension is still available we should provide documentation for it, although unfortunately Google has removed that documentation (of a still existing product of theirs).

I'm not sure if we should mention that she uses ChromeVox because officially she actually uses JAWS. :) I guess that's already confusing. But we had to opt for a screen reader that we can use in these circumstances.

Reviewer: Thanks, that makes sense 👍
